### ISSUE TITLE
Indicated required fields on new site form are incorrect

### ISSUE THREAD

**Issue by IanVS** [DefinedNet/mobile_nebula#202]
When creating a new site, the form appears to require a lighthouse/host, but does not.  On the other hand, it _does_ require a CA, but that is not indicated.

**Comment by jasikpark** [comment]
have we ended up with the "you can save an invalid site" state here? It seems like it'd still be useful to show the inputs that are required for a valid site as "required". Today, all you need to do to click "save" on a site is add a name, though it will have errors to fix when you open it.

**Comment by IanVS** [comment]
Yes, that is the desired state, that we can save an invalid form.  There are still "Needs attention" markers for those fields which are required for a functioning site, though.
